src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import core.basesyntax.enums.FigureType;
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final Random RANDOM = new Random();
```

Comment:
The `RANDOM` instance should not be public. Consider making it private to encapsulate the details of the class. Additionally, it's usually not a good practice to make mutable fields public static as it can lead to issues with thread safety and unexpected behavior when shared across multiple instances.